docs/templating/templates/view_matcher_reference.md

```diff
@@ -217,3 +222,40 @@ match:
 match:
     '@Ibexa\Contracts\ProductCatalog\ViewMatcher\LocationBased\RootLocation': true
 ```
+
+## Taxonomy entry ID
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id` matches based on an ID of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id': [1, 2, 3]'
+```
+
+## Taxonomy entry identifier
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier` matches based on an identifier of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier': ['spring', 'events', 'devices']
+```
+
+## Taxonomy entry level
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level` matches based on a level of the taxonomy entry.
+With this matcher, you can apply view rules based on a selection of taxonomy entry levels, by using the following logical operators: `<` , `>` , `<=`, `>=`.
```

Comment:
As there is no example for `'@@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': 2`, let suggest is equivalent `'@@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': '= 2'`.
```suggestion
With this matcher, you can apply view rules based on a selection of taxonomy entry levels, by using the following logical operators: `<` , `>` , `<=`, `>=`, `=`.
```